.github/workflows/check_tests.yml

```diff
@@ -5,400 +5,274 @@ on:
-            cargo-
       - name: cache sccache
         uses: actions/cache@v2
         continue-on-error: false
         with:
-          path: /home/runner/.cache/sccache
+          path: ~/.cache/sccache
           key: sccache-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: |
             sccache-
       - name: start sccache server
         run: sccache --start-server
       - name: init
+        shell: bash
         run: |
-          sudo apt update
-          sudo apt install -y pkg-config libssl-dev protobuf-compiler
+          apt update
+          apt install -y pkg-config libssl-dev protobuf-compiler curl build-essential clang git
           protoc --version
           curl -s https://sh.rustup.rs -sSf | sh -s -- -y
           source ${HOME}/.cargo/env
           rustup update
           rustup toolchain install nightly-2023-03-03
           rustup default nightly-2023-03-03
           rustup target add wasm32-unknown-unknown
+      - name: cache cargo
+        uses: Swatinem/rust-cache@v2
       - name: Run Integration Tests
+        shell: bash
         env:
           RUST_BACKTRACE: full
           RUSTC_WRAPPER: sccache
-          SCCACHE_CACHE_SIZE: 120G
-          SCCACHE_DIR: /home/runner/.cache/sccache
+          # SCCACHE_CACHE_SIZE: 120G
+          SCCACHE_CACHE_SIZE: 2G
+          SCCACHE_DIR: ~/.cache/sccache
         run: |
           source ${HOME}/.cargo/env
           RUSTC_BOOTSTRAP=1 cargo test -p integration-tests --release --features=calamari --no-default-features
           RUSTC_BOOTSTRAP=1 cargo test -p integration-tests --release --features=manta --no-default-features
       - name: stop sccache server
         run: sccache --stop-server || true
-  stop-integration-test-checks:
-    timeout-minutes: 15
-    needs: start-integration-test-checks
-    runs-on: ubuntu-20.04
-    if: ${{ always() }}
-    steps:
-      - continue-on-error: true
-        uses: audacious-network/aws-github-runner@v1.0.33
-        with:
-          mode: stop
-          github-token: ${{ secrets.SELF_HOSTED_RUNNER_TOKEN  }}
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: ${{ needs.start-integration-test-checks.outputs.aws-region }}
-          runner-label: ${{ needs.start-integration-test-checks.outputs.runner-label }}
-          aws-instance-id: ${{ needs.start-integration-test-checks.outputs.aws-instance-id }}
-      - name: discard stopper success/failure
-        run: true
   start-benchmark-checks:
     timeout-minutes: 180
-    runs-on: ubuntu-20.04
-    outputs:
-      runner-label: ${{ steps.start-self-hosted-runner.outputs.runner-label }}
-      aws-region: ${{ steps.start-self-hosted-runner.outputs.aws-region }}
-      aws-instance-id: ${{ steps.start-self-hosted-runner.outputs.aws-instance-id }}
+    runs-on: runtime
+    if: contains(github.event.pull_request.labels.*.name, 'A-benchmark-checks')
```

Comment:
please add the new labels to the PR description template

Reply:
done :)